assets/rpip-53/rewards-calculation-spec.md

```diff
@@ -0,0 +1,711 @@
+# Specification for Rewards Calculation
+
+This document serves as a formal specification for the way that the rewards intervals and the values within are calculated as part of the [Redstone](https://medium.com/rocket-pool/rocket-pool-the-merge-redstone-601d9efd6b4) rewards system.
+
+
+## Version
+
+This describes **v10** of the rewards calculation ruleset.
+
+This version implements [RPIP-53](https://github.com/rocket-pool/RPIPs/blob/rewards_v10/RPIPs/RPIP-53.md).
```

Comment:
will have to fix this link later. unless we make it relative here.

Reply:
Made it relative here.